crypto/sha/sha256.c

```diff
@@ -53,6 +54,13 @@ int SHA256_Init(SHA256_CTX *c)
     return 1;
 }
 
+int sha256_192_init(SHA256_CTX *c)
```

Comment:
This should be named `ossl_sha256_192_init`
We've adopted a naming convention whereby all exported but not public symbols start `ossl_`

Reply:
Just to be clear, the similar functions in `include/crypto/sha.h` predate this convention?

Reply:
Yes.